src/ServiceStack.OrmLite.Oracle.Tests/OracleParamTests.cs

```diff
@@ -269,328 +259,922 @@ public void ORA_ParamByteTest()
         }
 
         [Test]
-        public void ORA_ReservedNameComment_Test()
+        public void ORA_ReservedNameLevelAsAlias_Test()
         {
             using (var db = OpenDbConnection())
             {
                 DropAndCreateTables(db);
 
-                var row = new ParamComment { Comment = 1, Id = 2 };
+                var row = new ParamLevelAsAlias { Id = 2, ApprovalLevel = 212218 };
                 db.Insert(row);
 
-                row.Comment = 454;
+                row.ApprovalLevel = 676;
                 db.Update(row);
             }
         }
 
+        [ImportMany(typeof(IParam))]
+// ReSharper disable UnassignedField.Compiler
+        private IEnumerable<IParam> _reservedNameParams;
+// ReSharper restore UnassignedField.Compiler
 
         [Test]
-        public void ORA_ReservedNameLeft_Test()
+        public void ORA_ReservedNames_Test()
         {
+            ResolveReservedNameParameters();
             using (var db = OpenDbConnection())
             {
-                DropAndCreateTables(db);
+                foreach (var param in _reservedNameParams)
```

Comment:
If any of these fail, we won't know which one failed without using a debugger. You could use a try / catch that rethrows with the type of param, or use the TestCaseSource attribute to run the method for each param type.